packages/feedback-service/src/feedback-config/typedef.graphql

```diff
@@ -41,8 +41,9 @@ input FeedbackConfigInput {
 type Query {
   listFeedbackConfigs: [FeedbackConfigType]
   getFeedbackConfigById(id: ID!): FeedbackConfigType
-  getFeedbackConfigByAppId(appId: String): FeedbackConfigType
-  getFeedbackConfigByProjectId(projectId: String): FeedbackConfigType
+  getFeedbackConfigByAppId(appId: String!): FeedbackConfigType
```

Comment:
@deshmukhmayur @hybridx @ayushsatyam146 

How about join all this into one single query instead of three. Just queryByKey or something and then specify which type of key. Also this is starting to get really confusing

Reply:
@akhilmhdh this does make sense but we added this end point just as a `first-commit` kind of thing.